config.go

```diff
@@ -771,7 +772,7 @@ func LoadConfig(interceptor signal.Interceptor) (*Config, error) {
 		// If it's a parsing related error, then we'll return
 		// immediately, otherwise we can proceed as possibly the config
 		// file doesn't exist which is OK.
-		if _, ok := err.(*flags.IniError); ok {
+		if lnutils.ErrorAs[*flags.IniError](err) || lnutils.ErrorAs[*flags.Error](err) {
```

Comment:
I guess one thing that needs to be tested is if the comment above still stands true? Meaning, do we still ignore a non-existing file? Or is that returned as a `*flags.Error`?